code/modules/mob/living/carbon/human/species/_species.dm

```diff
@@ -375,7 +375,7 @@
 
 	switch(damagetype)
 		if(BRUTE)
-			var/damage_amount = damage * hit_percent * brute_mod * H.physiology.brute_mod
+			var/damage_amount = (damage/(1+(blocked/ARMOUR_CONSTANT))) * brute_mod * H.physiology.brute_mod
```

Comment:
this line is written multiple times just with different mods.
might it be good to make a define like
```
#define ARMOUR_EQUATION(damage, armour, modifier) (damage / (1 + (armour / ARMOUR_CONSTANT)) * modifier
```
and then pass the damage, blocked and mod1 * mod2 into the params.